js/src/forum/addSubscriptionFilter.js

```diff
@@ -8,24 +8,23 @@ export default function addSubscriptionFilter() {
     if (app.session.user) {
       const params = app.search.stickyParams();
 
-      params.filter = 'following';
-
       items.add('following', LinkButton.component({
-        href: app.route('index.filter', params),
-        children: app.translator.trans('flarum-subscriptions.forum.index.following_link'),
+        href: app.route('following', params),
         icon: 'fas fa-star'
-      }), 50);
+      }, app.translator.trans('flarum-subscriptions.forum.index.following_link')), 50);
     }
   });
 
-  extend(IndexPage.prototype, 'config', function () {
-    if (m.route() == "/following") {
+  override(IndexPage.prototype, 'setTitle', function (original) {
+    if (m.route.get().startsWith("/following")) {
```

Comment:
Can we use `this.attrs.routeName` or `app.current.get('routeName')` here, so that the code isn't tied to the path but only to the name?

Reply:
Ah, THAT was the one I wanted to change but forgot to...